TestApp/src/main/AndroidManifest.xml

```diff
@@ -46,6 +49,7 @@
         <service
             android:name="com.scribd.armadillo.download.DefaultExoplayerDownloadService"
             android:exported="false"
+            android:foregroundServiceType="dataSync|mediaPlayback"
```

Comment:
Is there a service within Armadillo that's used for playback, or is it only the service inside exoplayer?

Reply:
We do have `PlaybackService` but not sure if it requires `dataSync`

Reply:
Ah, it's already got `mediaPlayback`. All good, then.